models/concept.go

```diff
@@ -56,31 +58,13 @@ func (h Concept) RetriveAllBySourceId(sourceId int) ([]*Concept, error) {
 	var dataSourceModel = new(Source)
 	omopDataSource := dataSourceModel.GetDataSource(sourceId, Omop)
 
-	var concept []*Concept
+	var concepts []*Concept
 	result := omopDataSource.Db.Model(&Concept{}).
-		Select("concept_id, concept_name, domain.domain_id, domain.domain_name").
+		Select("concept_id, concept_name, domain.domain_id, domain.domain_name, concept_class_id, concept_class_id as concept_type").
```

Comment:
Is this intended?

```
concept_class_id, concept_class_id as concept_type
```

Reply:
it should not be there. I've removed it now (see last commit for details). 
The code is an attempt of using a custom gorm type to get our "Concept type" value. In this case, concept type is a subset of concept_class_id, so the custom gorm type needs this mapping to be in the query... Please look at the last changes and let me know what you think. The alternative is doing something like we already do for other derived fields like `NmissingRatio` ([see here](https://github.com/uc-cdis/cohort-middleware/blob/master/models/concept.go#L143-L155))...this part for  `NmissingRatio` results in some more code, but is perhaps less surprising/confusing...